go/database/mpt/io/live.go

```diff
@@ -74,6 +74,31 @@ func Export(ctx context.Context, directory string, out io.Writer) error {
 	}
 	defer db.Close()
 
+	return exportLive(ctx, db, out)
+}
+
+func ExportFromArchive(ctx context.Context, directory string, out io.Writer, block uint64) error {
```

Comment:
I think names `ExportArchive` and `ExportFromArchive` are too similar, I think this new method should be named perhaps  `ExportBlock`, `ExportBlockHeight`, `ExportBlockFromArchive`, etc.

Reply:
Im not really a fan of these names but I dont see a better alternative. So lets use these I guess.

Reply:
How about `ExportHead` instead of `ExportBlock`, then keep `ExportArchive` and use `ExportBlockFromArchive`?

Reply:
not sure what would be called `ExportHead` ?  Anyway: `ExportFromArchive -> ExportBlockFromArchive` sounds good to me.

Reply:
`ExportHead` would be `ExportLive` - exporting live genesis from `LiveDB`

Reply:
no, dont change this, it is unrelated to this PR, lets stay just with adding the method `ExportBlockFromArchive`

Reply:
ic. I thought you wanted to rename all of the `Export...` funcs. Understood now.